soc/riscv/nordic_nrf/common/vpr/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories(.)
+
+zephyr_library()
+zephyr_library_sources(soc_irq.S soc_irq.c vector.S)
+
+set(SOC_LINKER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/linker.ld CACHE INTERNAL "")
```

This `linker.ld` can be removed, if no changes to the common file are needed.
```suggestion
set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/riscv/common/linker.ld CACHE INTERNAL "")
```

---

done, thanks